### PR TITLE
[BIOIN-1751] Functions for working with flow based pileup

### PR DESCRIPTION
**Pull request by ilyasoifer**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [project]
 name = "ugbio_utils"
-version = "1.3.4-0dev-60"
+version = "1.3.4-0dev-64"
```
[the remark this answers is not in the document]
Comment (inbalzelig-ugi) [reply pyproject.toml:3]: Don't touch the version manually. 
We have an action that updates the version (for all members) on each PR.

**Comment by ilyasoifer** [comment]
> I wonder if this should be part of the core or if it can be a separate member. I'm not sure if there is a benefit either way, but it's worth thinking about.

Separate member with a separate docker? Why?

**Comment by inbalzelig-ugi** [comment]
> > I wonder if this should be part of the core or if it can be a separate member. I'm not sure if there is a benefit either way, but it's worth thinking about.
> 
> Separate member with a separate docker? Why?

It all depends on where it is used. I'm not familiar with the use case so I can't say.

**Review by TammushB — APPROVED** [review]

**Comment by ilyasoifer** [comment]
@inbalzelig-ugi - few questions
1. `uv.lock` seems to be changed, not sure why and if it is ok to commit it
2. all *toml files updated. Not sure if it is good or not
Thanks for advice

**Comment by inbalzelig-ugi** [comment]
> @inbalzelig-ugi - few questions
> 
> 1. `uv.lock` seems to be changed, not sure why and if it is ok to commit it
> 2. all *toml files updated automatically. Not sure if it is good or not
>    Thanks for advice

1. It is ok. It is also ok not to commit it. Don't worry about this file.
2. This is a limitation we have. We are still looking for a workaround to avoid this general update and update the version in one place.

**Pull request merged**